### PR TITLE
[Release 5.0.0] Drop iOS 9 and iOS 10, update example project

### PR DESCRIPTION
**Pull request by Kaspik**
What does this implement/fix? Explain your changes.
---------------------------------------------------
- drop support for iOS 9 and iOS 10, update layout to use safe area layout guide
- unify versioning in XCode
- update Example project to support iOS 11+ too
- update code conditions
- update podspec to support iOS 11+

Where has this been tested?
---------------------------
**Devices/Simulators:** iPhone 11 Pro

**iOS Version:** iOS 13.3

**Swift Version:** 5.1

In order to merge MessageKit PR https://github.com/MessageKit/MessageKit/pull/1261 we are going to drop also iOS 9 and iOS 10 for IBAV.

### CODE REVIEW

**Comment by nathantannar4** [comment]
Whats the benefit of dropping support for these two OS? These changes don't seem to add anything.

**Comment by Kaspik** [comment]
Yeah, I don't think they do. This was a request from MessageKit maintainer (@austinwright) to keep consistency between MessageKit and IBAV... It's not a blocker, so we can close this if you want.

File: Sources/InputBarAccessoryView.swift [hunk 1]
```diff
@@ -504,16 +494,14 @@ open class InputBarAccessoryView: UIView {
...
-            if let window = window {
-                guard window.safeAreaInsets.bottom > 0 else { return }
-                windowAnchor?.isActive = false
-                windowAnchor = contentView.bottomAnchor.constraint(lessThanOrEqualToSystemSpacingBelow: window.safeAreaLayoutGuide.bottomAnchor, multiplier: 1)
-                windowAnchor?.constant = -padding.bottom
-                windowAnchor?.priority = UILayoutPriority(rawValue: 750)
-                windowAnchor?.isActive = true
-                backgroundViewLayoutSet?.bottom?.constant = window.safeAreaInsets.bottom
-            }
+        if let window = window {
```
Comment (austinwright) [inline Sources/InputBarAccessoryView.swift:497]: can this be a guard instead?

File: Sources/Supporting Files/InputBarAccessoryView+Availability.swift [hunk 2]
```diff
@@ -55,12 +55,13 @@ extension AutocompleteManager {
     public var isCaseSensitive: Bool {
         get { return false }
         set {
-            if isCaseSensitive {
+            if newValue {
```
Comment (austinwright) [inline Sources/Supporting Files/InputBarAccessoryView+Availability.swift:58]: I think this change was merged to master on a different PR, can you bring this PR up to date?

**Comment by austinwright** [comment]
@nathantannar4, there are a few reasons for this. The main reason as Jakub pointed out was to keep consistency between MessageKit and IBAV. It also allows us to remove `#if available` conditions for iOS 10/11 (not much, I know, but it is still something). Another big thing (for me) that we found out the hard way, was that even if a dependency such as this one does not use location services: If the app does use location services elsewhere, and is linked against iOS 11 or higher, you are still subject to the app store rule that specifies you must provide the pre-iOS 11 usage key & description. There may be other pros and cons but these were the main drivers for me. Up to you if you want to merge this or not, like Jakub said it is not a blocker.

**Comment by nathantannar4** [comment]
Oh I didn’t know that. That’s an interesting side effect. Ok I think I’m ok with this then.

File: README.md [hunk 3]
```diff
@@ -30,13 +31,14 @@ pod 'InputBarAccessoryView', '4.2.2'
...
 github "nathantannar4/InputBarAccessoryView"
+
 # Swift 4.2
 github "nathantannar4/InputBarAccessoryView" "4.2.2"
 ```
 
 ### Requirements
 
-iOS 9.0+
+iOS 11.0+
```
Comment (nathantannar4) [inline README.md:41]: I would leave it as iOS 9+ but where we have the Swift 4 pod version, add an entry for pre iOS 11 so people could checkout the last compatible version.

**Comment by nathantannar4** [comment]
As a side note. Might want to consider holding off on the version bumping to a separate PR. That way the newer version could include other PRs that are open

**Comment by Kaspik** [comment]
- merged master
- updated PR comments
- removed app bump from podspec
- added noted about iOS 9 + iOS 10 latest version

**Pull request merged**